### PR TITLE
update ios showDialog method apprequests callback

### PR DESCRIPTION
**Pull request by taina0407**
Implement facebook sdk delegate to call callback for "apprequests" method

### CODE REVIEW

**Comment by xjunior** [comment]
:+1: please, release it!

File: src/ios/FacebookConnectPlugin.m [hunk 1]
```diff
@@ -310,10 +312,9 @@ - (void) showDialog:(CDVInvokedUrlCommand*)command
         content.recipients = params[@"to"];
         content.title = params[@"title"];
 
+        self.gameRequestDialogCallbackId = command.callbackId;
         dialog.content = content;
         [dialog show];
-        CDVPluginResult *pluginResult = [CDVPluginResult resultWithStatus:CDVCommandStatus_OK];
```
Comment (jeduan) [inline src/ios/FacebookConnectPlugin.m]: Can you rebase? Think this is a regression just fixed in master.

Comment (taina0407) [inline src/ios/FacebookConnectPlugin.m]: I've just rebase the pull request, please check it.

**Comment by jeduan** [comment]
Merged, thanks!

**Pull request merged**